mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -403,16 +438,15 @@ def evaluate(self,
                 f.write('name\n')
                 for name in names:
                     f.write(name + '\n')
-                f.close
+                f.close()
 
             eval_config = trackeval.Evaluator.get_default_eval_config()
-
+            gt_folder = osp.join(local_dir.name, tmp_prefix)
```

Comment:
Can we remove `tmp_prefix` and directly put gt files into `local_dir.name` directory?
It is unnecessary to have a `tmp_prefix` in the path of `gt_folder`.